Jenkinsfile.macos.electron

```diff
@@ -129,7 +129,7 @@ pipeline {
           }
 
           // this job is going to run on a macOS slave, which cannot use an instance-profile
```

Comment:
Ditto.
Also, we've been working hard to get away from master/slave, use controller/agent instead please.